### ISSUE TITLE
[Feature]: reverseKeyValue when object has multiple value.

### ISSUE THREAD

**Issue by ChanhyukPark-Tech** [toss/slash#235]
### **Package Scope**

- [x] Add to an existing package
- [ ] New package

Package name: [@toss/utils reverseKeyValue](https://slash.page/libraries/common/utils/src/reverseKeyValue.i18n)

### **Overview**

below is always correct
```tsx
expect(reverseKeyValue({ jbee: 'eebj' })).toEqual({ eebj: 'jbee' });
```

but how about this object which is multiple same value

```tsx
reverseKeyValue({ jbee: 'eebj',jbee2:'eebj' }) // { "eebj": "jbee2"} 
```

It is not bug,, but people can make human error I think

### **Describe the solution you'd like**

Why don't we put the same values together in an array?

```tsx
reverseKeyValue({ jbee: 'eebj',jbee2:'eebj' }) // { "eebj": ["jbee","jbee2"]} 
```

### **Additional context**

In the case above, please suggest if it can be handled in another good way👍

**Comment by wooki-kim** [comment]
안녕하세요 기여를 해보려고 구조를 파악중입니다.
질문이 있어요. 해당 파일의 유닛 테스트는 어떻게 해야할까요?
`yarn workspace @toss/utils test:unit` 명령어를 쓰니 아무런 반응이 없습니다. 혹시 제가 잘못 사용하고 있는지 궁금합니다.

**Comment by ChanhyukPark-Tech** [comment]
Thank you for your interest.
I am also Korean, but since the rules of this repository are to communicate in English, I will follow the rules and answer them in English.

Did you install the appropriate package through `yarn install`? 
I run the command you showed me(`yarn workspace @toss/utils test:unit`), and the test works normally. (From the root path of the project)

https://github.com/toss/slash/assets/69495129/6f2bf8a7-ebd5-475e-936d-7f67504112f7

**Comment by wooki-kim** [comment]
Thank you for your response and providing the information. Although the exact cause is unknown, it seems that the issue was resolved after upgrading Yarn.

Congratulations on submitting your first open-source PR! I have reviewed your PR at https://github.com/toss/slash/pull/250. I will provide feedback and suggestions for improvement promptly.

**Comment by wooki-kim** [comment]
Thank you for your response. I appreciate it.
After upgrading yarn and reconfiguring it with yarn berry, the issue has been resolved.
I have submitted my first open source PR. You can find it here: https://github.com/toss/slash/pull/250
I would greatly appreciate any feedback you can provide, and I will make the necessary revisions accordingly.

**Comment by ChanhyukPark-Tech** [comment]
I agree with this [https://github.com/toss/slash/pull/284#discussion_r1253089937](interpretation)

**Comment by ChanhyukPark-Tech** [comment]
@wooki-kim 
I've seen the code of your PR and it works well, but I don't think it's appropriate for the way `javascript` object works, what do you think? If you agree, I will close this issue

**Comment by ChanhyukPark-Tech** [comment]
thanks for involving this issue 
@wooki-kim @minsoo-web
